libs/design-system/src/components/EntityAvatar.tsx

```diff
@@ -34,7 +34,7 @@ export function EntityAvatar({
     />
   )
   const linkEl = href && (
-    <Link href={href} underline="none">
+    <Link href={href} underline="none" data-testid={`entity-link-${href}`}>
```

Comment:
This seems a bit redundant, we can give it a go but I would probably have tried to leverage the existing href in combination with a general testid. Combining more widely useful testids means we don't have to put a testid on specific elements for every test case.
```
page.getByTestId('entity-avatar').locator(`a[href*=${href}]`)
```

Reply:
I went ahead and adjusted this a bit. Instead of such a specific ref, I used "entity-link"--I see value in having these links instrumented, but I adjusted the selector in a similar way to what you suggested:

```ts
await page
    .locator(
      'a[data-testid="entity-link"][href*="8a03682f22857f306a95f55a28fa9edf"]'
    )
    .click()
```